draft-ietf-tls-wkech.xml

```diff
@@ -72,12 +72,16 @@
         <t>
             We define a well-known URI at which an HTTP origin can
             inform an authoritative DNS server, or other interested parties,
-            about this origin's Service Bindings, i.e. its "HTTPS" DNS records.
-            These instructions can include Encrypted ClientHello (ECH)
+            about this origin's Service Bindings, such as its "HTTPS" DNS records.
```

Comment:
I believe "i.e." is correct here, because this protocol is limited to HTTP origins.  As currently specified, it cannot produce other kinds of Service Bindings.

Reply:
If that's the case, the "such as" is redundant with "this origin's service bindings" phrase.  Fixup commit pushed.